test/gtest/diagnostics.cpp

```diff
@@ -69,51 +69,58 @@ TEST_F(Diagnostics, P4_16_Disable) {
         @diagnostic("uninitialized_out_param", "disable")
     )"));
     EXPECT_TRUE(test);
-    EXPECT_EQ(0u, ::P4::diagnosticCount());
+    EXPECT_EQ(0u, P4::diagnosticCount());
```

Comment:
Probably do not even need the `P4::` qualifier here. Not sure what our general policy is but these functions are all available in the P4 namespace.

Reply:
Right. I guess I am used to working on a code where `P4` is just one of the namespaces :-).